src/test/java/com/devcycle/sdk/server/local/DVCLocalClientTest.java

```diff
@@ -41,10 +41,13 @@ private static DVCLocalClient createClient(String config){
                 .build();
         DVCLocalClient client = new DVCLocalClient(apiKey, options);
         try {
-            // wait one second for the config to be loaded by the client
-            Thread.sleep(1000);
+            while(!client.isInitialized())
```

Comment:
Is there a default timeout on the test runner somewhere? Might be a little confusing to debug this otherwise if the tests just hang because the client never initializes.

Reply:
likely yeah, at minimum the GHA would timeout at some point. but that is a good point a potential infinite loop in the unit test is bad.  i'll fix that